quick_tour/the_big_picture.rst

```diff
@@ -155,11 +155,12 @@ because that will be explained in the next section)::
 
     use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
     use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+    use Symfony\Component\HttpFoundation\Request;
```

Comment:
You are right. I've added it to make it identical to the controller they will see. We decided to add the `Request` import to make it easier for newcomers: https://github.com/symfony/symfony-standard/blob/2.8/src/AppBundle/Controller/DefaultController.php#L7

However, the content of the `indexAction()` is not the same as in the symfony-standard. I did this because in this case it's more confusing than useful.

In any case, let me know what you think about this and I'll make the changes you say. Thanks.